src/ci/disable-py-cache.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+set -ex
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+
+sed -i "s/^from memoization import cached/##### from memoization import cached/" $(find . -name '*.py' -not -path .python_packages)
```

Comment:
(1) whatever we do here really would really benefit from docs that make it very clear what is going on
(2) what if, instead of rewriting every file that used memoization, we introduce a memoization module in `onefuzzlib` that could be programmatically disabled, or at worst, just rewritten in one place? IOW, in one mode, it would export a typed identity decorator, and in the other, the actually `memoization.cached` decorator?

Reply:
I am working on upstreaming a patch to python-memoization that will keep the type signatures for decorated functions, which will remove the need for this work-around.

I will add documentation to these scripts.  These are intended as a stop-gap only.

Reply:
Cool, this is okay a stopgap, and the comments will help.